src/Store/KeyBindingsStoreConnector.re

```diff
@@ -236,11 +238,7 @@ let start = () => {
           };
         };
 
-      Log.info(
-        "Loading "
-        ++ string_of_int(List.length(keyBindings))
-        ++ " keybindings",
-      );
+      Log.infof(m => m("Loading %i keybindings", List.length(keyBindings)));
 
       dispatch(Actions.KeyBindingsSet(defaultBindings @ keyBindings));
```

Comment:
This might be a candidate for removal though. But not sure how significant key keybinding count is.

Reply:
We should keep this, because we don't get a count from the keybindings - the count of keybindings is actual helpful for drilling into issues like https://github.com/onivim/oni2/pull/1164 (ie, were we able to load any bindings)